### ISSUE TITLE
Need a user-mod directory for the SMYLE simulations for ndep...

### ISSUE THREAD

**Issue by ekluzek** [ESCOMP/POP2-CESM#42]
I need a user-mod directory for the CESM2.1.x series for POP for the SMYLE simulations. I think the only thing I need for POP is to point to an ndep file that includes the historical period as well as the future scenario so you can run from 1850-2025.

The changes needed are documented here:

https://docs.google.com/document/d/1rV3fWCIt-D1fAbzQ6uaQeZyLagzzKnopofr88eXegzY

The CESM issue for this is here...

https://github.com/ESCOMP/CESM/issues/178

Who can I work with for this?

**Comment by mnlevy1981** [comment]
I'm probably the right person to talk to about it, but POP doesn't really use user-mod directories outside of testing. Could we set up a time to chat about what the user-mod directory would do and we can try to get the same functionality with different methods?

**Comment by ekluzek** [comment]
Yes, that's what @klindsay28 pointed out to us. I think the way forward is for us to work with @klindsay28 and then he might reach out to you for @mnlevy1981 to create a tag with the needed changes. @klindsay28 does that sound right to you?

A drive for the smyle project is here:

https://drive.google.com/drive/folders/1_x751KiGUO47aRdEwrPVyFAuISfmQGta
